packages/app/src/components/Crashes/errorMessage.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-rainbow-components';
+import { ErrorContainer, ErrorText, Title, SecondTitle } from './styled';
+import CrashImg from './crashImg';
+
+const ErrorMessage = ({ errorTrace }) => {
+    return (
+        <ErrorContainer>
```

Comment:
by default this container will take 100% of parent width and height, then, when using it in the RainbowFirebaseApp pass the component prop with this same ErrorMessage but with a wrapper that set `height: 100vh;`